go/mysql/flavor_mysql.go

```diff
@@ -413,26 +413,31 @@ const BaseShowTables = `SELECT t.table_name,
 `
 
 // TablesWithSize80 is a query to select table along with size for mysql 8.0
-//
-// Note the following:
-//   - We use a single query to fetch both partitioned and non-partitioned tables. This is because
-//     accessing `information_schema.innodb_tablespaces` is expensive on servers with many tablespaces,
-//     and every query that loads the table needs to perform full table scans on it. Doing a single
-//     table scan is more efficient than doing more than one.
-//   - We utilize `INFORMATION_SCHEMA`.`TABLES`.`CREATE_OPTIONS` column to do early pruning before the JOIN.
-//   - `TABLES`.`TABLE_NAME` has `utf8mb4_0900_ai_ci` collation.  `INNODB_TABLESPACES`.`NAME` has `utf8mb3_general_ci`.
-//     We normalize the collation to get better query performance (we force the casting at the time of our choosing)
 const TablesWithSize80 = `SELECT t.table_name,
 		t.table_type,
```

Comment:
these comments were helpful.
Can you add the comments with new changes to know why it is done this way.

Reply:
Added useful comments :)